site/docs/master/release.md

```diff
@@ -6,38 +6,25 @@
    As of the time of writing, the version is defined in `pkg/buildinfo/version.go`.
 
 ## Kubernetes release
-Is this release for a Kubernetes release?  If so, apply the following steps:
+Is there a Kubernetes release since the last Sonobuoy release? If so, apply the following steps:
 
 ### Conformance images
 Ensure the upstream conformance script is working appropriately:
-  * Build the kind images for this new version.
-    1. Checkout K8s locally at the tag in question
-    1. Run `make check-kind-env` to ensure the repo/tag are correct
-    1. Run `make kind_images`
-    1. Run `make push_kind_images`
   * Update the `kind-config.yaml` file with the new image version [here](https://github.com/vmware-tanzu/sonobuoy/blob/master/kind-config.yaml).
+  * Run conformance tests
 
 ### Setup e2e image registry for new version
-Add the new list of E2E test images for the targed version.
+We have [opened an ongoing issue upstream](https://github.com/kubernetes/kubernetes/issues/96475) to make this more elegant.
+Until then, ensuring the registry list did not grow from previous release is somewhat manual.
```

Comment:
~grow~ -> change

Reply:
it's intentional to say "addition", because we shouldn't subtract in case any of them got deleted for backwards compatibility reasons. for example, if v1.20 -> v1.21 drops one of the registries, we probably want to keep our copy complete to make sure Sonobuoy still supports overriding v1.20?